docs/signature_auth.md

```diff
@@ -0,0 +1,11 @@
+# Signature authentication 
+
+
+To limit who can input data into your endpoints, you can optionally create a kubernetes secret such that only POST, PUT or PATCH requests with the correct signature in its HTTP header will be allowed through. If this optional kubernetes secret is not given, or is removed, all requests, regardless of type are automatically allowed by default.
+
+To enable this signature verification, create a kubernetes opaque/generic secret with name `signature-secret` and password being the secret of your choice.
```

Comment:
The secret is called `${NAME}-authentication` and contains a key `signature-secret`